index.rst

```diff
@@ -1,143 +1,63 @@
 meet all its requirements within 3 seconds of ending a short
 slew (3.5 degrees in 2 seconds)
 
-*The requirement is for the system to have settled to the same (within accuracy) position after a short slew, within 3 seconds.*
-
-Test Case
----------
-`LVV-11258 <https://github.com/lsst-sitcom/notebooks_vandv/tree/tickets/SITCOM-798/notebooks/tel_and_site/subsys_req_ver/m1m3>`__ 
-
-Plot the settling time of the M1M3 for X, Y, Z, RX, RY, and RZ.
-
-The data comes from the EFD: imsData. The IMS is the
-Independent Measurement System, a set of electronic
-micrometers that measure the displacement of the M1M3 mirror
-with respect to the cell. According to LTS-88 it has a 4 um
-accuracy in XYZ and 3e-5 degree accuracy in RXRYRZ.
-
-Test Data
----------
-*Looked for suitable cases in Rubin TV. TBD a systematic search for those cases closest to the requirement settings (3.5 degree slew at full speed, not reached yet as far as I know as of 240823).*
-
-- dayObs = 2023-06-27
-- seqNo = 450
-- Duration = 5s
-- Azimuth only 3.5 degree slew
-
-See `RubinTV <https://roundtable.lsst.codes/rubintv-dev/summit/tma/historical/2023-06-27>`__
-
-Results
--------
-IMS XYZ position with azimuth and elevation reference. Vertical line denotes reference time (slew stop):
-
-
-.. figure:: /_static/xyz_vs_azel.png
-   :name: fig-xyzvsazel
-
-   IMS XYZ during the slew, compared to azimuth and elevation from mount information. 
-
-RXRYRZ rotation with azimuth and elevation reference. Vertical line denotes reference time (slew stop):
-
-.. figure:: /_static/rxryrz_vs_azel.png
-   :name: fig-rxryrzvsazel
-
-   IMS RXRYRZ during the slew, compared to azimuth and elevation from mount information.
-
-Settling behavior from the IMS measurements. IMS values during and after slew, with RMS behavior with respect to end of the plot, with a 3 second requirement window .
-
-.. figure:: /_static/xsettle.png
-   :name: fig-xsettle
-
-   IMS x residual compared to value at slew stop, RMS, in mm.
-
-.. figure:: /_static/ysettle.png
-   :name: fig-ysettle
-
-   IMS y residual compared to value at slew stop, RMS, in mm.
-
-.. figure:: /_static/zsettle.png
-   :name: fig-zsettle
-
-   IMS z residual compared to value at slew stop, RMS, in mm.
-
-.. figure:: /_static/xrotsettle.png
-   :name: fig-xrotsettle
-
-   IMS rotation in x residual compared to value at slew stop, RMS, in degrees.
-
-.. figure:: /_static/yrotsettle.png
-   :name: fig-yrotsettle
-
-   IMS rotation in y residual compared to value at slew stop, RMS, in degrees.
-
-.. figure:: /_static/zrotsettle.png
-   :name: fig-zrotsettle
-
-   IMS rotation in z residual compared to value at slew stop, RMS, in degrees.
-
-
+**LTS-88-REQ-0052**: The positioning system SHALL maintain mirror decenter less than
++/- 6 micrometer, mirror tilt less than +/- 24 e-6 degree, and piston less than +/- 1
+micrometer, all relative to the mirror cell, after a slew of 3.5 degrees or less (short slew).
```

Comment:
Although Req-051 and 052 are indeed valid requirements for the dynamical tests, they are not the only ones. If you look at [sitcomtn-084](https://sitcomtn-084.lsst.io) content, you will see it also needs to comply with Req-053 and Req-054 on top of the Req-052. So, my comment is to call your attention to keeping track of the requirements listed in each technote. Maybe a good idea would be to list requirements and link them with the corresponding technote, like a summary relating a specific test (technote) with its requirement.

Reply:
So, maybe instead of linking all the requirements, we should add a link to https://ls.st/LTS-88. Hopefully, Systems Engineering can help us map the requirements. However, it might be easier to highlight specific reqs in each of the more focused technotes.